### ISSUE TITLE
Embedded videos need metadata

### ISSUE THREAD

**Issue by mbakeryo** [mlibrary/heliotrope#1754]
The title should be there (hidden) before a user
gets to the video player.

**Comment by conorom** [comment]
@jmcglone Here is a nice summary of this kind of thing:
https://dequeuniversity.com/tips/provide-iframe-titles

Can you confirm that:
1) the `<title>` of the embedded page is what we're targeting here, so for now I'm just going to set that to the FileSet title
2) the iframe itself should have a title attribute as well

**Comment by conorom** [comment]
Will create a PR following the advice in that link.
